src/homework2/task2.py

```diff
@@ -0,0 +1,33 @@
+"""Найти самое длинное слово в введенном предложении. В случае если их
+    несколько, самое левое в строке. Учтите что в предложении есть знаки
+    препинания.
+"""
+
+
+def longest_word(str_):
+    """Поиск самого длинного слова в предложении.
+
+    :param str_: входная строка
+    :return: строка. Самое длинное слово в предложении (в случае если их
+        несколько, самое левое в строке).
+        в случае если
+    """
+    # write your code here
+    znaki = '.?!:;*=/|%()_-+{}[]<>~№` '
+    for smv in znaki:  # проходимся по строке
```

Comment:
O(m*n) solution. m - number of marks, n - length of the string. Can be solved in O(n)